types/lib/model.d.ts

```diff
@@ -634,17 +634,15 @@ export interface CountOptions<TAttributes = any>
 /**
  * Options for Model.count when GROUP BY is used
  */
-export interface CountWithOptions<TAttributes = any> extends CountOptions<TAttributes> {
-  /**
-   * GROUP BY in sql
-   * Used in conjunction with `attributes`.
-   * @see Projectable
-   */
-  group: GroupOption;
-}
+export type CountWithOptions<TAttributes = any> = SetRequired<CountOptions<TAttributes>, 'group'>
```

Comment:
Changing `interface` to `type`.
This might be a type breaking change if someone extended the interface.

However, I think the type is better presented with a `type` instead of `interface`